pkg/test/resources/samples/meshhttproute_samples.go

```diff
@@ -0,0 +1,75 @@
+package samples
+
+import (
+	plugins_xds "github.com/kumahq/kuma/pkg/plugins/policies/core/xds"
+	meshhttproute_api "github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute/api/v1alpha1"
+	meshhttproute_xds "github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute/xds"
+	"github.com/kumahq/kuma/pkg/util/pointer"
+	envoy_common "github.com/kumahq/kuma/pkg/xds/envoy"
+)
+
+func MeshHttpOutboundRouteWithSeveralRoutes(serviceName string) *meshhttproute_xds.HttpOutboundRouteConfigurer {
```

Comment:
nit: do we need the word `Route` twice in this name? Should it just be `MeshHttpOutboundWithSeveralRoutes`?